README.md

```diff
@@ -1,70 +1,47 @@
-# Getting Started with Create React App
+# City-Weather App
+A react-redux web app that help users to search for weather condition of any city, view location & date, view wind status & wind direction, view humidity, air pressure, view weather condition of the next 5 days, and convert from degree Celsius to Fahrenheit or vice visa
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+### Live Demo
```

Comment:
- [ ] [REQUIRED] Kindly ensure that this project is been deployed using Heroku, Netlify, or GitHub Pages as stated in the project requirement.